native/src/main/java/io/ballerina/stdlib/sql/nativeimpl/QueryProcessor.java

```diff
@@ -51,6 +54,11 @@
  */
 public class QueryProcessor {
 
+    private static TransactionResourceManager trxResourceManager = null;
+    private static Connection connection = null;
+    private static PreparedStatement statement = null;
+    private static String sqlQuery = null;
```

Comment:
Aren't these used only within the method? Why is this initiated at class level?

Reply:
They are all initialized within `initQuery()` but are used inside both `query()`  and `queryRow()`

Reply:
Fixed in 18e9282 by removing initQuery() `method`.